shell/mixins/compact-input.ts

```diff
@@ -1,7 +1,5 @@
-import { createApp } from 'vue';
-const vueApp = createApp({});
 
-export default Vue.extend({
+export default {
```

Comment:
If we were previously using `Vue.extend`, we would now want to use `defineComponent`

Reply:
NVM `defineComponent` is used primarily for components and not mixins.

Reply:
Spoke offline